### ISSUE TITLE
As an editor, I want to add a timeline or map to an edition

### ISSUE THREAD

**Issue by jamiefolsom** [SHOREdevelopers/COVE#48]

**Comment by daverett** [comment]
See [this comment](https://github.com/SHOREdevelopers/COVE/issues/52#issuecomment-248062568) to #52

**Comment by jamiefolsom** [comment]
@daverett OK, so only admins can manage the contents of an edition? That's fine with me if it is with you and Dino and Amanda. 

Couple of follow-up questions: 
- Is that currently the case, that admins can manage edition contents?
- Should we open a testable user story which documents how that work is done?
- Or if not, will this work be done by you after launch? Or by Dino or someone else?

Thanks

**Comment by daverett** [comment]
I'm thinking through this from a slightly different angle. Since we have no way, according to our current integration plan, to identify a team at COVE Studio (AS) with a group at The COVE, the only content that an individual could collect as an Edition would be his or her own; that is, assuming that we have a way to publish documents from AS to the COVE. Since edition creation is a team effort, no single user would be able to create a full edition, unless either 1) the "group" of users carried over from the AS to the COVE or 2) a user had free-reign to add any document within the COVE to his or her edition, which seems a non-starter. That seems, to me, to be the biggest barrier to offering a coherent "edition builder" tool. Secondary concerns, which are also very important, have to do with the ordering of documents within an edition and the amount of metadata that would be required to model that. Say, for example, that an editor wanted two paratextual documents to fall in a particular order, a "bibliography" and an "editorial methodology". Said editor might want to one document to appear before another on a ToC or menu. I can provide that level of idiosyncratic re-ordering to users, but when packaged all together, it would seem to be a lot of work on the end user side, when coupled with other forms of metadata related to indexing in NINES/ARC as well. 

In other words, the architecture of an "edition" can be a complicated and multifarious thing that I'm thinking, at the moment, is best left to folks who understand the metadata at the COVE.

Finally, I can provide a testable user story for creating an edition out of multiple docs at the COVE, but there is a learning curve there.  And I would do this work, completely independent of the SOW, at least to start.

**Comment by jamiefolsom** [comment]
OK, sounds like there's a further conversation to be had about the edition builder. Let's take this up at our tech check-in.
